commands/list_releases.go

```diff
@@ -221,19 +211,27 @@ func listReleases(args listReleasesArguments) (listReleasesResult, error) {
 		return result, microerror.Mask(unknownError)
 	}
 
-	// sort releases by date
+	// sort releases by version (descending)
 	if len(releasesResponse) > 1 {
 		slice.Sort(releasesResponse[:], func(i, j int) bool {
-			return releasesResponse[i].Timestamp > releasesResponse[j].Timestamp
+			vi := semver.New(releasesResponse[i].Version)
+			vj := semver.New(releasesResponse[j].Version)
+			return vj.LessThan(*vi)
 		})
 	}
 
 	// sort changelog and components by component name
 	for n := range releasesResponse {
 		slice.Sort(releasesResponse[n].Components[:], func(i, j int) bool {
+			if releasesResponse[n].Components[i].Name == "kubernetes" {
+				return true
+			}
 			return releasesResponse[n].Components[i].Name < releasesResponse[n].Components[j].Name
```

Comment:
This is a hack to always make the `kubernetes` component appear in the front of the list.